Quiz.py

```diff
@@ -0,0 +1,24 @@
+class Question:
+     def __init__(self, prompt, answer):
+          self.prompt = prompt
+          self.answer = answer
+
+question_prompts = [
+     "Who Invented Python \n(a) Guido van Rossum\n(b)Dennis Ritchie",
+     "Which Year Python Was Realesed ?\n(a) 1991\n(b)1996",
+]
+
+questions = [

```

Comment:
Storing the answers separately from the questions is not a very maintainable design.  Probably keep the questions and the answers together in one place, ideally so that the question encodes the correct answer in a way which is clear to the developer but obscured from the user.

Reply:
hey Thanks For that Information , I am Actually new in GIT so, I don't Know much about it but thanks again for that info I will Try to do something about it . >>>